adding  & updating Readme (#169)
ash-sxn/GSoC-2023-docker-based-quickstart: README.md

    * `python` - 04_python-tutorial => `docker compose up -d python`
    * `node` - 05_nodejs => `docker compose up -d node`
* If no argument is used i.e. `docker compose up -d`, it runs the latest default example.

> * The above command uses prebuild images, which is not always secure to pull for unknow source.

Reviewer: ```suggestion
* The above command utilizes prebuilt images, which you might want to be cautious about as their source is not currently known or part of the Jenkins CI organization.
```

* So If you want to build images yourself add `-f build-docker-compose.yaml` after `docker compose`

> * eventually to make the command look something liks this `docker compose -f build-docker-compose.yaml up -d node` for build the Node Tutorial.

Reviewer: ```suggestion
* Eventually, the command should resemble something like this: `docker compose -f build-docker-compose.yaml up -d node` to build the Node Tutorial.
```
